certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -2318,10 +2367,117 @@ PROCEDURE EXPIREORDELETEXFERS
            SET EXPIRATIONDATE TO SYSTEMDATE
           END
         END
+       BNODEBUGMSG="SHARE XFER TYPE 2 DELETE/EXPIRE FMERROR='"+FMERROR+"'"
```

Comment:
Do you want to check for `FMERROR<>""` before outputting this debug?

Reply:
Nope. I like the idea of having the statement print for each FM attempt.  An " FMERROR='' " tells me a lot when tracing for errors